src/index.html

```diff
@@ -7,12 +7,52 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Calendar</title>
+
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/main.scss"
+    />
```

Comment:
The stylesheet link references a `.scss` file. SCSS files need to be compiled to CSS before they can be linked in an HTML document. Ensure that `styles/main.scss` is compiled to a `.css` file and update the link accordingly.